AirCastingTests/Logic/Statistics/MeasurementsStatisticsControllerTests.swift

```diff
@@ -9,18 +9,18 @@ class MeasurementsStatisticsControllerTests: XCTestCase {
     let dataSourceMock = DataSourceMock()
```

Comment:
please remove changes to this file from this PR since it doesn't change anything besides some random whitespaces